ps.go

```diff
@@ -43,6 +43,9 @@ var psCommand = cli.Command{
 		}
 
 		psArgs := context.Args().Get(1)
+		if psArgs == "--" {
+			psArgs = context.Args().Get(2)
+		}
```

Comment:
Yes, it is another thing need to be fixed, -- to let "runc ps" support multiple ps arguments.
I'll fix it on top of this patch.